app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -94,4 +94,7 @@ CONFIG_PROBE=y
 CONFIG_PROBE_DMA_MAX=2
 CONFIG_KCPS_DYNAMIC_CLOCK_CONTROL=y
 CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING=y
+CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING_NUM_CHANNELS=2
+CONFIG_GOOGLE_RTC_AUDIO_PROCESSING_MIC_BITS=32
+
```

Comment:
should no space line?

Reply:
Please review in #8571 . This PR is just a place to deliver the same code against mtl-007-drop-stable.

Reply:
Oh, actually this patch got dropped from that branch somehow!  Will fix.  But when I do please review there :)